src/controllers/Book.js

```diff
@@ -0,0 +1,90 @@
+const bcrypt = require('bcrypt-nodejs');
+const TokeController = require('../assets/Token');
+const Book = require('../models/Book');
+
+module.exports = {
+  async index(req, res) {
+    const books = await Book.findAll();
+
+    return res.json(books);
+  },
+
+  async store(req, res) {
+    const id_user = req.userId;
+    const {
+      name,
+      author,
+      rating,
+      description,
+      intention,
+      value,
+      image
+    } = req.body;
+
+    try {  
+      const book = await Book.create({
+        name,
+        author,
+        rating,
+        description,
+        intention,
+        value,
+        image,
+        id_user
+      });
+
+      if(!book)
+        return res.status(400).json({ error: 'Unable to register book' });
+
+        return res.json({
+          book
+        });
+
+    } catch (err) {
+      console.log(err);
+      return res.status(500).json({ error: 'Internal Server Error!'});
+    }
+  },
+
+  async update(req, res) {
+    const { id } = req.params;
+
+    try {
+      const updated = await Book.update(req.body, { where: { id } });
+
+      if (updated) {
+        const book = await Book.findOne({ where: { id } });
```

Comment:
this is unnecessary, the const updated already contains the info about the book updated, you can just do the lines 58 and 59 with the const updated

Reply:
We did the test removing this, but without success, since that kept the select